data/cameras.xml

```diff
@@ -8520,6 +8520,19 @@
 		<Crop x="4" y="0" width="-52" height="0"/>
 		<Sensor black="1024" white="16383"/>
 	</Camera>
+	<Camera make="FUJIFILM" model="X100F">
+		<ID make="Fujifilm" model="X100F">Fujifilm X100F</ID>
+		<CFA2 width="6" height="6">
+			<ColorRow y="0">RBGBRG</ColorRow>
+			<ColorRow y="1">GGRGGB</ColorRow>
+			<ColorRow y="2">GGBGGR</ColorRow>
+			<ColorRow y="3">BRGRBG</ColorRow>
+			<ColorRow y="4">GGBGGR</ColorRow>
+			<ColorRow y="5">GGRGGB</ColorRow>
+		</CFA2>
+		<Crop x="0" y="0" width="-126" height="0"/>
+		<Sensor black="1024" white="16383"/>
```

Comment:
@dtorop 
Re black/white points: all the decoders need to be extended to read all of this info from the raw files themselves.
cameras.xml 
E.g. In this case: https://raw.pixls.us/getfile.php/1166/exiftool/DSCF0150.RAF.exiftool.txt
```
[RAF]           XTransLayout                    : BGGRGG RGGBGG GBRGRB RGGBGG BGGRGG GRBGBR
[RAF]           BlackLevel                      : 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022 1022
```
I'd guess the CFA is specified for crop
```
[MakerNotes]    RawImageWidth                   : 6032
[MakerNotes]    RawImageHeight                  : 4032
```
while the full image is
```
[RAF]           RawImageFullWidth               : 6160
[RAF]           RawImageFullHeight              : 4032
```